apps/ui/components/application/ApprovedReservations.tsx

```diff
@@ -367,18 +496,35 @@ function createReservationUnitLink({
 }): JSX.Element {
   const { pk } = reservationUnit;
   const name = getTranslationSafe(reservationUnit, "name", lang);
-  return pk != null && pk > 0 ? (
-    <IconButton400
+  if (pk == null || pk <= 0) {
+    return <span>{name}</span>;
+  }
+  return (
+    <ReservationUnitLink
       href={getReservationUnitPath(pk)}
       label={name}
       openInNewTab
-      icon={<IconLinkExternal aria-hidden />}
+      icon={<IconLinkExternal aria-hidden="true" />}
     />
-  ) : (
-    <span>{name}</span>
   );
 }
 
+const CancelButton = styled(Button).attrs({
+  theme: "black",
+  variant: "supplementary",
+  iconLeft: <IconCross aria-hidden="true" />,
+})`
+  white-space: nowrap;
+`;
+
+const StyledStatusLabel = styled(StatusLabel)`
+  @media (width <= ${breakpoints.s}) {
```

Comment:
This seems to me like it would only take effect if viewport width is smaller than (or equal to) our "mobile" breakpoint, so the specified styles wouldn't work in e.g. a 321px wide view. Is this by design/correct?

I'd use `width < ${breakpoints.m}` - assuming we intend to cover mobile use cases (== everything up to the medium/"tablet"-breakpoint)?

Reply:
it's correct.

mobile breakpoint here is `breakpoints.s`: `576px`

Reply:
then again maybe it looks better if we break at `m: "768px"`

I'll change it and make it a constant (so it's consistent in all places).